README.md

```diff
@@ -45,9 +45,14 @@ modules:
       - "Download the latest version here"
       tls_config:
         insecure_skip_verify: false
+      protocol: "tcp" # accepts "tcp/tcp4/tcp6", defaults to "tcp"
+      preferred_ip_protocol: "ip4" # preferred in case of "protocol: tcp", accepts "ip4/ip6", defaults to "ip6"
```

Comment:
This is a bit wordy, considering the rest is self-explanatory. Can it be trimmed?